middleware/sentry.go

```diff
@@ -122,3 +122,39 @@ var SentryErrorHandler = func(res *http.Response, url string) error {
 
 	return nil
 }
+
+func GetSentryErrorHandler(filters ...func(res *http.Response, url string) bool) func(res *http.Response, url string) error {
```

Comment:
so you can include multiple `GetSentryErrorHandler(OkStatusFilter, BadRequestFilter)` 👍